File: 프로그래머스/unrated/181914. 9로 나눈 나머지/9로 나눈 나머지.java
import java.util.*;
import java.math.*;

class Solution {
    public int solution(String number) {
        String number1 = "123";
        BigInteger answer = new BigInteger(number);
        System.out.println(answer);
        BigInteger remain = new BigInteger("9");
        answer = answer.remainder(remain);
        
        int i = answer.intValue();
        return i;
    }
}

File: 프로그래머스/0/181914. 9로 나눈 나머지/9로 나눈 나머지.py
# 음이 아닌 정수를 9로 나눈 나머지 
# 그 정수의 각 자리 숫자의 합을 9로 나눈 나머지
def solution(number):
    answer = 0
    for i in str(number):
        print(i)
        answer += int(i) % 9
    answer %= 9
    return answer